### ISSUE TITLE
[dotnet] Add option to update all nuget packages.

### ISSUE THREAD

**Issue by AraHaan** [dotnet/sdk#11582]
I would like if the dotnet cli program that I can invoke ``dotnet`` had a way to like ``dotnet nuget update --all-packages`` where it would update:

1. All packages in packages.config at once.
2. All packages in ``PackageReference`` at once on all projects in a solution file (if there is a solution file) otherwise all projects in the current directory and subdirectories.

This would make it helpful for those who just want to update the packages without waiting for VS2019 to resolve them (sometimes can be slow for some people) or those who does not want to update them by gui.

**Comment by AraHaan** [comment]
Also it seems the package manager console has this like so: ``Update-Package -ProjectName MyProject -reinstall`` However this is not feasible because eventually I will need this to make my own github action that will create a commit that updates all dependencies at once without removing anything that the ``PackageReference``s has in ``Condition`` statements.

**Comment by gitfool** [comment]
@AraHaan I agree this functionality would be nice to have integrated into the official `dotnet nuget`. Meanwhile, have you tried using [dotnet-outdated](https://github.com/jerriep/dotnet-outdated)?

**Comment by AraHaan** [comment]
oh I did not know that tool is a thing, it will preserve all of the ``Condition`` statements that dependabot & VS2019 preserves right?
